api.bs

```diff
@@ -1248,6 +1512,33 @@ returning an [=epoch index=]:
 
 </div>
 
+### Safety Limits Configuration ### {#safety-limits-configuration}
+
+[=User agents=] configure [=safety limits=] by defining the following values:
+
+*   <dfn>Global budget per epoch</dfn> (&epsilon;<sub>global</sub>):
+    The maximum privacy budget available across all [=sites=] per [=epoch=],
+    specified in [=microepsilons=].
+
+*   <dfn>Impression site quota per epoch</dfn> (&epsilon;<sub>imp-quota</sub>):
+    The maximum privacy budget that a single [=impression site=]
+    can enable to be consumed from the [=global privacy budget=] per [=epoch=],
+    specified in [=microepsilons=].
+
+*   <dfn>Conversion site quota per epoch</dfn> (&epsilon;<sub>conv-quota</sub>):
+    The maximum privacy budget that a single [=conversion site=]
+    can consume from the [=global privacy budget=] per [=epoch=],
+    specified in [=microepsilons=].
+
+*   <dfn>Quota count cap</dfn> (<var>k</var><sub>quota-count</sub>):
+    The maximum number of distinct [=sites=]
+    that can create new quota budgets
+    within a single [=user action context=].
+
+<p class=note>Typical values might be:
+TODO
```

Comment:
In #174, I've added a section for implementation-defined limits.  I think you could add to that section for these.  In particular, you want to set these as multiples of the per-site epsilon budget, so you could set minimum values for them.